File: python/pytorch/train/vgg16/train_tiny.py
from python.pytorch.models.vgg16 import run_experiment

if __name__ == "__main__":
    run_experiment(
        dataset_path="data/tiny_imagenet_png",
        output_file="vgg16_tiny.csv",
        checkpoint_path="checkpoints/vgg16_tiny.pth",
        img_size=(32, 32),
        grayscale=False,
        test_split="val"
    )
    
#PYTHONPATH=. python3 python/pytorch/train/vgg16/train_tiny.py

File: R/train/vgg/train_tiny.r
source("R/models/vgg16.r")
cat("Script avviato!\n")
run_experiment(
    dataset_path = dataset_path,
    checkpoint_path = file.path(root, "R/checkpoints/vgg16_tiny_r.pt"),
    img_size = c(32, 32), # forziamo 32x32 anche per Tiny (consistenza)
    grayscale = FALSE,
    test_split = "val", # Tiny usa 'val' come split
    epochs = 30,
    batch_size = 128
)